sdk/storage/azure-storage-blob/azure/storage/blob/_lease.py

```diff
@@ -79,13 +73,13 @@ def acquire(self, lease_duration=-1, **kwargs):
             (-1) for a lease that never expires. A non-infinite lease can be
             between 15 and 60 seconds. A lease duration cannot be changed
             using renew or change. Default is -1 (infinite lease).
-        :keyword ~datetime.datetime if_modified_since:
+        :keyword datetime if_modified_since:
```

Comment:
nit: I think we should leave the `~datetime` in all these. I believe it helps the auto-generated docs actually link to the `datetime` docs.